h2o-extensions/target-encoder/src/main/java/ai/h2o/targetencoding/TargetEncoderMojoWriter.java

```diff
@@ -45,18 +44,17 @@ protected void writeExtraInfo() {
    * Writes target encoding's extra info
    */
   private void writeTargetEncodingInfo() throws IOException {
-    TargetEncoderOutput output = ((TargetEncoderModel) model)._output;
-    TargetEncoderModel.TargetEncoderParameters teParams = output._parms;
+    TargetEncoderOutput output = model._output;
+    TargetEncoderParameters teParams = output._parms;
     writekv("with_blending", teParams._blending);
     if (teParams._blending) {
       writekv("inflection_point", teParams._inflection_point);
       writekv("smoothing", teParams._smoothing);
     }
-    writekv("priorMean", output._prior_mean);
     
     Map<String, Boolean> col2HasNAs = output._te_column_to_hasNAs;
```

Comment:
This could be simplified to a set ? If the column is in the set, it has NAs. We're talking about the original code here, so let's leave it. Just making some notes.

Reply:
Vecs have NA counts in rollups available, not sure if this is even required.

Reply:
good tip, I agree that a set would be better here: although it needs to be changed carefully as afaik the mojo stores this map (also not optimally) as a list of kv pairs.
Will add a comment and try to chnage it in a future PR.

Reply:
Thank you so much. That's as much as I was hoping for, nothing more. I owe you one :croissant: .